dotmanager/utils.py

```diff
@@ -290,3 +290,7 @@ def print_success(message: str) -> None:
 def is_dynamic_file(target: Path) -> bool:
     """Returns if a given path is a dynamic file"""
     return os.path.dirname(os.path.dirname(target)) == normpath("data")
+
+def find_files(filename: str, paths: List[str]):
```

Comment:
Typehint for return value is missing. Also `paths: List[str]` should be `paths: List[RelPath]`.